Sources/Driver/Process+Extensions.swift

```diff
@@ -0,0 +1,95 @@
+import Foundation
+
+extension Pipe {
+
+  /// Returns the contents decoded as UTF-8, while consuming `self`.
+  public func readUTF8() -> String {
+    String(decoding: fileHandleForReading.readDataToEndOfFile(), as: UTF8.self)
+  }
+
+}
+
+extension Process {
+
+  /// The textual output of a process run.
+  public typealias OutputStreams = (standardOutput: Pipe, standardError: Pipe)
+
+  /// The results of a process run that exited with a nonzero code.
```

Comment:
```suggestion
  /// The results of a process run that exited with a non-zero code.
```
nit: Native English speakers will know better whether or not the dash is necessary but I think the form employed should be consistent with the name of the type. `NonZeroExit` suggests that "non" and "zero" are separate.

Reply:
Good point!  But I think the name should be changed to `NonzeroExit` to address it; “nonzero” [is a word](https://www.merriam-webster.com/dictionary/nonzero#:~:text=1,nonzero%20affixes).